packages/react-search-ui-views/src/Autocomplete.tsx

```diff
@@ -109,15 +128,15 @@ function Autocomplete({
           !!autocompletedResults &&
           autocompletedResults.length > 0 && (
             <ul className="sui-search-box__results-list">
-              {autocompletedResults.map(result => {
+              {autocompletedResults.map((result) => {
                 index++;
-                const titleSnippet = getSnippet(
-                  result,
-                  autocompleteResults.titleField
-                );
-                const titleRaw = getRaw(result, autocompleteResults.titleField);
+                const titleField =
+                  typeof autocompleteResults === "boolean"
```

Comment:
with autocompleteResults configuration could be either a boolean or a configuration object, i needed to derive the type by doing this primitive boolean check.